### DISCUSSION TITLE
Mount Point URL

### DISCUSSION THREAD

**Discussion by Tullstar** [AzuraCast/AzuraCast#4779] [Q&A]
Perhaps I am missing something but have not gotten the custom mount point to function as I had hoped.

In order to not change/update all our radio links if we move to using Azuracast, I need to match our existing links.  What Azuracast assigned and I have working is: https://example.org/radio/8100/kwoj.mp3 but what I need to create is: https://example.org/kwoj.mp3  

Is that possible either in an Azuracast setting or on the server?  Thanks for any help.

**Comment by stevenaj** [comment]
I have just seen your post and it is almost identical to the post I made last night.  Azuracast don't think its a bug as my issue was converted to a discussion point under #4931 

If anyone knows how to achieve this then I would also like to hear from them as we do not want to update all our current stream url's as this will cause some major headaches for us.

It is a shame as this has now become a showstopper for our migration to Azuracast.
